src/assets/text/referencesText.js

```diff
@@ -82,10 +82,10 @@ export default {
     references: [
```

Comment:
Is it necessary that we divide up the references by text, quote and image?

Reply:
It's not necessary, but I like it separated out personally--it was especially important to me when there were also image references (there was no other way to easily find where the images came from). Now there are no image references, so it's less of an issue so I can combine them.